tests/components/overseerr/conftest.py

```diff
@@ -0,0 +1,67 @@
+"""Common fixtures for the Overseerr tests."""
+from collections.abc import Generator
+from unittest.mock import AsyncMock, patch
+
+from overseerr_api.models import RequestCountGet200Response
+import pytest
+
+from homeassistant.components.overseerr.const import DEFAULT_URL, DOMAIN
+from homeassistant.const import CONF_API_KEY, CONF_URL
+from homeassistant.core import HomeAssistant
+
+from tests.common import MockConfigEntry
+
+
+@pytest.fixture(name="config_entry")
+def fixture_config_entry() -> MockConfigEntry:
+    """Return the default mocked config entry."""
+    return MockConfigEntry(
+        domain=DOMAIN,
+        unique_id="12345",
+        data={CONF_URL: DEFAULT_URL, CONF_API_KEY: "test-api-key"},
+        version=1,
+    )
+
+
+@pytest.fixture
+def mock_setup_entry() -> Generator[AsyncMock, None, None]:
+    """Override async_setup_entry."""
+    with patch(
+        "homeassistant.components.overseerr.async_setup_entry", return_value=True
+    ) as mock_setup_entry:
+        yield mock_setup_entry
```

Comment:
I don't think that's a good idea, because we'd want to see that that method actually succeeds/fails. Rather patch away your downstream lib, so the `async_setup_entry` is actually executed